vim.exe.stackdump

```diff
@@ -0,0 +1,19 @@
+Stack trace:

```

Comment:
Is this file needed? :D

Reply:
I have no idea where this came from.
Removed